descriptors/ModuleDescriptor-template.json

```diff
@@ -536,6 +536,32 @@
         }
       ]
     },
+    {
+      "id": "inventory-update-ownership",
+      "version": "0.1",
+      "handlers": [
+        {
+          "methods": ["POST"],
+          "pathPattern": "/inventory/items/update-ownership",
+          "permissionsRequired": ["inventory.items.update-ownership.item.post"],
+          "modulePermissions": [
+            "inventory-storage.items.item.put",
```

Comment:
We will need post permissions for creating items on the target tenant, and delete permissions for deleting them on the source tenant. Not sure yet if put permissions will be necessary

Reply:
Done, Thank you